packages/pocl-test-file-generator/src/generateRecord.js

```diff
@@ -0,0 +1,71 @@
+import moment from 'moment'
+
+const getRandomElementFromArray = arr => arr[Math.floor(Math.random() * arr.length)]
+
+const randomInteger = (min, max) => {
+  return Math.floor(Math.random() * (max - min + 1)) + min;
+}
+
+const JUNIOR_MIN_AGE = 13
+const JUNIOR_MAX_AGE = 16
+const SENIOR_MIN_AGE = 65
+
+export const AGE_RANGE = Object.freeze({
+  JUNIOR: 1,
+  ADULT: 2,
+  SENIOR: 3
+})
+
+const getRandomDateOfBirth = ageRange => {
+  const minAge = minAndMaxAgesForRange[ageRange].min
+  const maxAge = minAndMaxAgesForRange[ageRange].max
+  return moment().subtract(randomInteger(minAge, maxAge), 'years').subtract(1, 'day')  
+}
+
+const minAndMaxAgesForRange = Object.freeze({
+  [AGE_RANGE.JUNIOR]: Object.freeze({ min: JUNIOR_MIN_AGE, max: JUNIOR_MAX_AGE }),
```

Comment:
Another thought - I think freeze only works in strict mode, and we get strict mode implicitly by using the nodule system. have you actually tried it?

Reply:
Just tried this in `generateRecord.test.js`:

  it('freezes value of AGE_RANGE', () => {
    expect(() => {
      AGE_RANGE.ADULT = 4
    }).to.throw()
  });

and it passes (and fails when we're not using Object.freeze)